samples/monitor/E3_Monitor/__init__.py

```diff
@@ -0,0 +1,34 @@
+import azure.durable_functions as df
+from datetime import timedelta
+from typing import Dict
+
+def orchestrator_function(context: df.DurableOrchestrationContext):
+
+    monitoring_request: Dict[str, str] = context.get_input()
+    repo_url: str = monitoring_request["repo"]
+    phone: str = monitoring_request["phone"]
+
+    # Expiration of the repo monitoring
+    expiry_time = context.current_utc_datetime + timedelta(minutes=5)
+    while context.current_utc_datetime < expiry_time:
+        # Count the number of issues in the repo (the GitHub API caps at 30 issues per page)
```

Comment:
The context of the 30 issues per page probably isn't relevant to this piece of code. Though it would be nice to make the cap configurable as a parameter to the orchestration.

Reply:
Unfortunately the cap cannot be extended beyond 30 issues and I don't see the point in making it lesser. Personally, I would prefer to keep the comment in there just for completeness' sake. If someone decides to experiment with the sample by monitoring a "real" repo, then they might eventually hit the cap and get confused.

I don't feel strongly though.

Reply:
I was more referring to the fact that the actual check for how many issues lies in the activity function, and so it is more relevant to document that limitation in the activity (which you already did).

It's fine documenting it here, but it might confuse people a bit without context.